datalake-analytics-new-datalake-store/azuredeploy.json

```diff
@@ -0,0 +1,59 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "adlAnalyticsName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the Data Lake Analytics account to create."
+            }
+        },
+        "adlStoreName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the Data Lake Store account to create."
+            }
+        }
+    },
+	"variables": {
+		"defaultApiVersion": "2015-06-15",
+        "adlsApiVersion": "2015-10-01-preview",
+        "adlaApiVersion": "2015-10-01-preview"
+	},
+    "resources": [
+        {
+            "name": "[parameters('adlStoreName')]",
+            "type": "Microsoft.DataLakeStore/accounts",
+            "location": "East US 2",
```

Comment:
Is Data Lake limited to this region? Otherwise make this a variable with value = resourceGroup().location or a user input parameter with allowed values